example/src/main/java/org/hibernate/example/reactive/Main.java

```diff
@@ -79,7 +79,7 @@ public static void main(String[] args) {
 		)
 				.toCompletableFuture().join();
 
-		System.exit(0);
+		sessionFactory.close();
```

Comment:
whoa, why did you have a Sytem.exit there? :)

Reply:
'Cos I had no way to shut down Vert.x before :-)

Reply:
that's what I suspected.. so you already had a clue that something was leaking :)

Reply:
C'mon, all good Java programs call `System.exit()`, right @emmanuelbernard?

Reply:
now you're both in that club! :-)

Reply:
> C'mon, all good Java programs call `System.exit()`, right @emmanuelbernard?

It's an underrated API.